pyam/core.py

```diff
@@ -510,14 +517,125 @@ def check_aggregate(self, variable, components=None, units=None,
                                        **kwargs)]
 
         if len(diff):
-            msg = '{} of {} data points are not aggregates of components'
-            logger().info(msg.format(len(diff), len(df_variable)))
+            msg = '{} - {} of {} data points are not aggregates of components'
+            logger().info(msg.format(variable, len(diff), len(df_variable)))
 
             if exclude_on_fail:
                 self._exclude_on_fail(diff.index.droplevel([2, 3]))
 
+            diff = pd.concat([diff], keys=[variable], names=['variable'])
+
             return diff.unstack().rename_axis(None, axis=1)
 
+    def check_aggregate_regions(self, variable, region='World',
+                                components=None, units=None,
+                                exclude_on_fail=False, multiplier=1, **kwargs):
+        """Check whether the world timeseries data match the aggregation
+        of regions
+
+        Parameters
+        ----------
+        variable: str
+            variable to be checked for matching aggregation of regional data
+        region: str
+            region to be checked for matching aggregation of regional data
+        components: list of str, default None
+            list of regions, defaults to all regions except region
+        units: str or list of str, default None
+            filter variable and components for given unit(s)
+        exclude_on_fail: boolean, default False
+            flag scenarios failing validation as `exclude: True`
+        multiplier: number, default 1
```

Comment:
The multiplier is sometimes useful when comparing across variables when the variables are naturally in different units (e.g., carbon vs. carbon dioxide, EJ vs. GWa). I don't think that there is a use case over regions.